tasks/docs/serve.js

```diff
@@ -25,6 +26,8 @@ var
     assets      = require('../build/assets')
 ;
 
+let config = configConst;
```

Comment:
What's the purpose of this change?
I understand config cant be a constant because of later changes, so why not remove the declaration from the const group and instead? Would be more clean, as otherwise "configSetup" is never used anywhere else
```
let config      = require('../config/docs');
```

Reply:
in https://github.com/fomantic/Fomantic-UI/pull/2589/commits/f9cd84c64e2e751f7bcb1bed72da84cedd413670, in 2 files, `config` is assigned later in the script, so I fixed the problem quickly like I fixed it...

If I did not overlook something, `require('tasks/config/project/config').addDerivedValues()` can mutate the passed object, so now I fixed it properly in https://github.com/fomantic/Fomantic-UI/pull/2589/commits/bacb5cfedbb66f5f52875eaa2d67da0174e5a309, please review carefully